gluten-ut/common/src/test/scala/org/apache/spark/sql/GlutenTestsTrait.scala

```diff
@@ -130,21 +129,17 @@ trait GlutenTestsTrait extends GlutenTestsCommonTrait {
       expression: => Expression,
       expected: Any,
       inputRow: InternalRow = EmptyRow): Unit = {
-    val resolver = ResolveTimeZone
-    val expr = resolver.resolveTimeZones(expression)
-    assert(expr.resolved)
 
     if (canConvertToDataFrame(inputRow)) {
+      val resolver = ResolveTimeZone
+      val expr = resolver.resolveTimeZones(expression)
+      assert(expr.resolved)
+
       glutenCheckExpression(expr, expected, inputRow)
     } else {
-      logWarning(s"The status of this unit test is not guaranteed.")
-      val catalystValue = CatalystTypeConverters.convertToCatalyst(expected)
-      checkEvaluationWithoutCodegen(expr, catalystValue, inputRow)
-      checkEvaluationWithMutableProjection(expr, catalystValue, inputRow)
-      if (GenerateUnsafeProjection.canSupport(expr.dataType)) {
-        checkEvaluationWithUnsafeProjection(expr, catalystValue, inputRow)
-      }
-      checkEvaluationWithOptimization(expr, catalystValue, inputRow)
+      logWarning(
+        "Skipping evaluation - Expression could not be converted " +
```

Comment:
Suggestion:

Skipping evaluation - Nonempty inputRow cannot be converted to DataFrame due to **